Autopilot/Inc/TelemDriver.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef TELEM_DRIVER_HPP
+#define TELEM_DRIVER_HPP
+
+
+#include <ctime>
+#include <cstdint>
+
+struct TelemToPMData {
+    struct TelemToPMData { //Coming to telem from ground base.
+        float gpsLattitude, gpsLongitude; //Commanded gps position
+        float heading, distance //Commanded heading/distance
+        float roll, pitch, yaw;	//Commanded orientation (radians)
+        bool beginLand; //Command to start landing process
+        bool beginTakeoff; //Starting command
+        bool disconnectAutopilot; //Command to switch off autopilot
+    };
+};
+
+class TelemManager { //Pulling data from ground
```

Comment:
The telemetry manager class will be a state machine, just like the path and attitude manager classes. We should talk some more about what that means for your implementation.